akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -770,4 +770,26 @@ public void illustrateSourceQueue() throws Exception {
       }
     };
   }
+
+  @Test
+  public void illustrateSourceActorRef() throws Exception {
+    new TestKit(system) {
+      {
+        // #source-actorRef
+        int bufferSize = 10;
+
+        Source<Integer, ActorRef> source =
+            Source.actorRef(
+                bufferSize, OverflowStrategy.dropHead()); // note: backpressure is not supported
```

Comment:
Perhaps: 'if you need backpressure look at `Source.queue` instead`? (might be a bit long for on one line so perhaps put it above)

Reply:
It's already documented in the `stream-integrations.md` importing this example